dev-qt/qtbase/metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>qt@gentoo.org</email>
+		<name>Gentoo Qt Project</name>
+	</maintainer>
+	<use>
+		<flag name="concurrent">Build Qt6Concurrent module</flag>
+		<flag name="dbus">Build Qt6DBus module</flag>
+		<flag name="egl">Enable EGL integration</flag>
+		<flag name="eglfs">Build the EGL Full Screen/Single Surface platform plugin</flag>
+		<flag name="evdev">Enable support for input devices via evdev</flag>
+		<flag name="gssapi">Enable support for GSSAPI (<pkg>virtual/krb5</pkg>)</flag>
+		<flag name="gtk">Build the GTK platform theme plugin</flag>
+		<flag name="gui">Build Qt6Gui module</flag>
+		<flag name="ico">Enable ico image support</flag>
```

Comment:
It does seem like, in the sources, that it could be optional but indeed, not sure what happened to actually using it.

Indeed, could enable it unconditionally (and gif too).

Reply:
Done.